changelog/@unreleased/pr-191.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
+  description: '`optional<binary>` now represented in go as `*ReadCloser` and a received 204 status code is treated as nil'
```

Comment:
Can you expand this description further? Since these are rolled into the user-facing release notes, would like to make it easier to understand the change based on this content.

For example, something like:

```
Fixes issue where generated Conjure clients did not properly handle empty responses from Conjure endpoints returning `optional<binary>`.

The Conjure specification states that endpoints returning `optional<binary>` can return a 204 status code to indicate an empty optional (to distinguish from the case where the response is a non-empty optional with 0 bytes), but this was not previously supported by the clients generated by conjure-go. This change interprets `optional<binary>` as `*ReadCloser` and returns a `nil` value when the response is a 204.
```

Doesn't have to be this exact wording, but wanted to provide a concrete example of the level of detail that I think would be helpful for developers reading through the release notes to understand the changes.